app/src/main/java/ar/com/wolox/android/cookbook/room/RoomRecipePresenter.kt

```diff
@@ -0,0 +1,85 @@
+package ar.com.wolox.android.cookbook.room
+
+import android.app.Application
+import android.os.Handler
+import ar.com.wolox.android.cookbook.room.database.RoomDataEntity
+import ar.com.wolox.android.cookbook.room.database.RoomDatabaseManager
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import javax.inject.Inject
+
+class RoomRecipePresenter @Inject constructor(
+    val application: Application
+) : BasePresenter<RoomRecipeView>() {
+
+    private lateinit var db: RoomDatabaseManager
```

Comment:
Well, yes, it' okey. But there're some ways to do it better with services and injections. Maybe you could talk with Flor or with Doni that they're doing it on StayQuit.